cardano-node-chairman/cardano-node-chairman.cabal

```diff
@@ -10,22 +10,54 @@ license-files:         LICENSE
                        NOTICE
 build-type:            Simple
 
-library
```

Comment:
Previously the source files were under here, but hedgehog can't find them, so I converted this into a `common` module.